File: cmsong111/42748.py
def solution(array, commands):
    answer = []
    
    for i in commands:
        temp = array[i[0]-1:i[1]]
        temp.sort()
        answer.append(temp[i[2]-1])

        
    return answer

#https://programmers.co.kr/learn/courses/30/lessons/42748

array = [1, 5, 2, 6, 3, 7, 4]
commands =[[2, 5, 3], [4, 4, 1], [1, 7, 3]]
solution(array, commands)

File: sudhdkso/42748.cpp
#include <string>
#include <vector>
#include <algorithm>
#include <iostream>
using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
    vector<int> answer;
 
    for (int i = 0;i < commands.size();i++) {
        int Fidx = commands[i][0] - 1;
        int Lidx = commands[i][1] - 1;
        int idx = commands[i][2]-1;
        vector <int> a;
        for (int j = Fidx;j < Lidx + 1;j++)
            a.push_back(array[j]);

        sort(a.begin(), a.end());

        answer.push_back(a[idx]);
    }

    return answer;
}